cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -327,6 +329,8 @@ func (m *AzureManager) getFilteredScaleSets(filter []labelAutoDiscoveryConfig) (
 				klog.Warningf("ignoring vmss %q because of invalid minimum size specified for vmss: %s", *scaleSet.Name, err)
 				continue
 			}
+		} else if cfgMinSize >= 0 {
```

Comment:
Should we invert the precedence here and have the previous min/max foo (`scaleSet.Tags["min"]` and `scaleSet.Tags["max"]` have the higher assignment precedence? I think that might be a better back-compat story.

Reply:
I'm not sure I follow. We will read `scaleSet.Tags["min"]` (which is the current behaviour), and then `else if` that is not defined, we will check to see if there is a default `cfgMinSize` (new behaviour), and then if neither are defined we will ignore the vmss (current behaviour).

So if a default is not specified, then the current behaviour will be maintained. And the explicit tags will always be given the higher assignment precedence since even if the default is specified, we will still use the the explicit tag value

Reply:
Ah yes my bad, thanks!